cpp/dolfin/function/FunctionSpace.h

```diff
@@ -93,9 +93,24 @@ class FunctionSpace
 
   /// Interpolate expression into function space, returning the vector
   /// of expansion coefficients
+  /// @cond Work around doxygen bug for std::function
   /// @param[in] expansion_coefficients The expansion coefficients
   /// @param[in] f The function to be interpolated
-  void interpolate(Eigen::Ref<Eigen::Matrix<PetscScalar, Eigen::Dynamic, 1>>
+  void interpolate(
+      Eigen::Ref<Eigen::Matrix<PetscScalar, Eigen::Dynamic, 1>>
+          expansion_coefficients,
+      const std::function<Eigen::Array<PetscScalar, Eigen::Dynamic,
+                                       Eigen::Dynamic, Eigen::RowMajor>(
+          const Eigen::Ref<const Eigen::Array<
+              double, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>>&)>& f)
+      const;
+  /// @endcond
+
+  /// Interpolate expression into function space, returning the vector
+  /// of expansion coefficients
+  /// @param[in] expansion_coefficients The expansion coefficients
+  /// @param[in] f The function to be interpolated
+  void interpolate_c(Eigen::Ref<Eigen::Matrix<PetscScalar, Eigen::Dynamic, 1>>
```

Comment:
What does _c stands for? Not very clear.

Reply:
The idea is that it's not really in the public interface. The `_c` was added to indicate this, alluding to its use for interfacing with compiled C functions coming from Numba/CFFI.